### PR TITLE
Create maven.yml

### PR DESCRIPTION
**Pull request by Mathiasbrix**

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
```
Comment (semol18) [inline .github/workflows/maven.yml:24]: the path to the pom.xml is incorrect, it should read "run: mvn -B package --file JHotDraw/pom.xml".

**Review by semol18 — CHANGES_REQUESTED** [review]
Change the path in the maven.yml to the proper pom.xml file.
Like this "JHotDraw/pom.xml"

**Comment by aleksander-GD** [comment]
`name: Java CI with Maven

on:
  push:
    branches: [ master]
  pull_request:
    branches: [ master]

jobs:
  build:
    runs-on: windows-latest
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2
      - name: Set up JDK 1.8
        uses: actions/setup-java@v1
        with:
          java-version: 1.8
      - name: Cache the Maven packages to speed up build
        uses: actions/cache@v1
        with:
          path: ~/.m2
          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
          restore-keys: ${{ runner.os }}-m2

      - name: Build with Maven
        run: mvn -B package --file JHotDraw/pom.xml`

runs-on needs both ubuntu.

**Comment by Mathiasbrix** [comment]
The file has already been added to the master branch.

**Pull request closed without merging**